benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -32,9 +33,40 @@ static constexpr uint32_t kMaxSphereInstanceCount  = 3000;
 static constexpr uint32_t kSeed                    = 89977;
 static constexpr uint32_t kMaxFullscreenQuadsCount = 1000;
 
+static constexpr float4   kDefaultDrawCallColor        = float4(1.0f, 0.175f, 0.365f, 0.5f);
```

Comment:
why is the alpha to 0.5? (the original shader also had it to 0.5, but unsure if that's by design or an oversight)

Reply:
it is by design so that when the alpha blending is enabled, there is no need to change the shader const. this value is not doing anything if alpha blend is disabled.

Reply:
Right, makes sense, thanks!